### DISCUSSION TITLE
Embedding atproto posts in websites?

### DISCUSSION THREAD

**Discussion by naim94a** [bluesky-social/atproto#2277] [How to do it?] [answered]
Social media sites usually have some kind of snippet that allows embedding a post in websites, blogs, etc...
Is there a way to do that with atproto posts yet?

**Comment by bnewbold** [comment, marked as answer]
Yeah! This is very much on our radar and something we want to do for app.bsky posts.

We need to ensure that what we build makes sense long-term: we will basically need to support whatever embed system indefinitely. We may also want to use a different domain/hostname for the embed content; this is a common pattern for other large embed systems, possibly to reduce domain-block breakage, or for privacy/security concerns, or just general rate-limits and performance. For example, an embedded post on a popular news website would have a very different traffic and access pattern than a post viewed on bsky.app. We probably should support the OEmbed standard for easily creating such embeds, for example in Wordpress. And lastly this would probably be a centralized system and there might be ways to mitigate or reduce that effect.

None of these are blockers, and we should be able to get this done before too long, just need to get around to it.

**Reply by PSingletary** [reply]
As you are working towards this goal, I think a distinction should be made between ["Discourage apps from showing my account to logged-out users" web/app toggle]
(https://blueskyweb.zendesk.com/hc/en-us/articles/15835264007693-Data-Privacy) and a new option for "Allow embeds" in the UI
